sox.features.js

```diff
@@ -181,19 +181,17 @@
       // Description: For highlighting the names of answerers on comments
 
       function color() {
-        const answerCells = [...document.getElementsByClassName('answercell')];
-        for (let i = 0; i < answerCells.length; i++) {
-          const cell = answerCells[i];
-          const answererID = cell.querySelector('.post-signature:nth-last-of-type(1) a').href.match(/\d+/)[0];
-          const comments = document.getElementsByClassName('comments')[i + 1];
-          const commentUsers = $(comments).find('.comment-user');
-
-          for (let c = 0; c < commentUsers.length; c++) {
-            if (commentUsers[c].href.contains(`users/${answererID}`)) {
-              commentUsers[c].classList.add('sox-answerer');
-            }
-          }
-        }
+        for (let i = 0; i < $('.answercell').length; i++) {
+          const $this = document.getElementsByClassName('answercell')[i];
```

Comment:
This looks like it will end up querying the DOM for all the `.answercell` elements twice per iteration, once for the length condition, and once for the `$this` declaration.

For this part, I think we can just use the code that was there previously:

```
function color() {
  const answerCells = [...document.getElementsByClassName('answercell')];
  for (let i = 0; i < answerCells.length; i++) {
    const cell = answerCells[i];	
    const answerer = cell.querySelector('.post-signature:nth-last-of-type(1) a');
    if (!answerer) continue;

    // ... your changes here
  }
}

Reply:
Fine. I will go with that!